lighthouse/datadog_checks/lighthouse/data/conf.yaml.example

```diff
@@ -8,17 +8,32 @@ instances:
     #
   - min_collection_interval: 60
 
-    ## @param url - string - required
-    ## url of the website to get lighthouse metrics of
+    ## @param url - string - optional
+    ## url of the website to get lighthouse metrics of. This option is deprecated.
     ## ex: "https://www.datadoghq.com"
     #
     url: <URL_PLACEHOLDER>
 
+    ## @param urls - list of strings - required
+    ## urls of the websites to get lighthouse metrics of
+    ## ex: "https://www.datadoghq.com"
+    #
+    urls:
```

Comment:
Have you tried creating multiple check instances, each with one url? This should allow you to check multiple urls with one agent, unless there is a specific limitation with the check. It is  a smoother user experience to provide multiple urls in one instance, though.

Reply:
This can be a future improvement. I'll try to keep as much of the contributor's content the same for this PR since the maintainer had approved this already.